src/screens/BookmarkScreen.js

```diff
@@ -15,8 +15,8 @@ import {
 import { colors, consts, device, normalize, texts } from '../config';
 import { getKeyFromTypeAndCategory } from '../helpers';
 import { useBookmarks, useMatomoTrackScreenView } from '../hooks';
+import { useNewsCategories } from '../hooks/NewsCategories';
```

Comment:
could be an import from /hooks directly, right?

Reply:
done. (as in the other file in which it is used.)